blockprotocol/blockprotocol
# Add verification badges to `Hub`

## 🌟 What is the purpose of this PR?



This PR adds block verification badges to the hub page and to block pages.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203179076056209/1203623307402534/f) _(internal)_
- [Asana task](https://app.asana.com/0/0/1203966325298247/f) _(internal)_

## 🔍 What does this change?




- Added `verified: true` to local blocks;
- Display the badge if `verified` is set to `true` in the hub and block page.

## ❓ How to test this?



To test local blocks:
1.  Run `yarn workspace @apps/site exe scripts/prepare-blocks.ts`;
2. Check if blocks in the hub are displaying the badge.

To test published blocks:
1. Publish a random block (message me via slack if you need help with doing this locally);
2. Add a `verified: true` to it directly in MongoDB;
3. Check if blocks in the hub are displaying the badge.

## 📹 Demo


![image](https://user-images.githubusercontent.com/37453800/218326911-33f5a56d-57b8-4515-9765-768a2199aba8.png)

![image](https://user-images.githubusercontent.com/37453800/218447846-1b214b34-7f9b-40ac-98e7-8a696f36d558.png)
